CPS-????/README.md

```diff
@@ -0,0 +1,96 @@
+---
+CPS: ?
+Title: Light Client Protocols for Cardano
```

Comment:
```suggestion
Title: Light Client Protocols
```
Audience for CIP/CPS is entirely focused on Cardano, so the phrase `for Cardano` is not necessary & has generally been removed for other documents in the repository.